packages/swap/src/event-handlers/networkEgressScheduled.ts

```diff
@@ -22,13 +22,20 @@ export default async function networkEgressScheduled({
 }: EventHandlerArgs): Promise<void> {
   const { id, asset, amount } = eventArgs.parse(event.args);
 
-  await prisma.egress.create({
-    data: {
+  await prisma.egress.upsert({
```

Comment:
there will be two egress scheduled events?

Reply:
No there will be 1 egress for 2 swaps